src/index.css

```diff
@@ -2,4 +2,163 @@
   margin: 0;
   padding: 0;
   box-sizing: border-box;
+  list-style-type: none;
+  text-decoration: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+body {
+  font-family: "Arial Narrow Bold", sans-serif;
+  width: 100%;
+}
+
+.header {
+  width: 100%;
+  height: 5.5rem;
+  background: #fff;
+}
+
+.header-container {
+  width: 50%;
+  margin: 0 20px;
+  height: 100%;
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+}
+
+.title {
+  color: #0290ff;
+  font-size: 1.875rem;
+}
+
+.nav {
+  width: 60%;
+  display: flex;
```

Comment:
**Changes Required**
- [ ] The `display` property and value `flex`  is over repeated. I suggest you have a class say `.flex`  in your `.jsx` files and in your `.css` files give it a property of display and value of flex. In this way, you will just declare it in one place and it will make your code more readable.